Lib/glyphsLib/builder/custom_params.py

```diff
@@ -0,0 +1,188 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import (print_function, division, absolute_import,
+                        unicode_literals)
+
+import re
+
+from glyphsLib.util import bin_to_int_list
+from .filters import parse_glyphs_filter
+from .constants import GLYPHS_PREFIX, PUBLIC_PREFIX, CODEPAGE_RANGES, \
+    UFO2FT_FILTERS_KEY
```

Comment:
let's stick to either parentheses or backslashes for multi-line import statements. I prefer the former